tests/zfs-tests/tests/functional/direct/dio.kshlib

```diff
@@ -190,17 +167,13 @@ function verify_dio_write_count #pool bs size mnpnt
 
 	log_note "Checking for $dio_wr_expected Direct I/O writes"
 
-	prev_dio_wr=$(get_iostats_stat $pool direct_write_count)
+	prev_dio_wr=$(kstat iostats.direct_write_count $pool)
 	dio_and_verify write $size $bs $mntpnt "sync"
-	curr_dio_wr=$(get_iostats_stat $pool direct_write_count)
+	curr_dio_wr=$(kstat iostats.direct_write_count $pool)
 	dio_wr_actual=$((curr_dio_wr - prev_dio_wr))
 
 	if [[ $dio_wr_actual -lt $dio_wr_expected ]]; then
-		if is_linux; then
-			cat /proc/spl/kstat/zfs/$pool/iostats
-		else
-			sysctl kstat.zfs.$pool.misc.iostats
-		fi
+		kstat iostats $pool
```

Comment:
Shouldn't this (and few similar below) be group?

Reply:
Ahh yes, good catch. They're dumping state on the error path, so I didn't see them.